src/Components/Nav/Nav.js

```diff
@@ -33,29 +36,37 @@ function Nav() {
               <MenuButtons path="/alram" name="알람" />
             </NavList>
           </NavUl>
-          <LoginSignButton
-            onClick={() => setIsLoginModal(!isLoginModal)}
-            isActiveMain={isMainPage}
-          >
-            로그인
-            <span></span>
-            회원가입
-          </LoginSignButton>
+          {localStorage.getItem('KakaoToken') && isLoginActive === true ? (
```

Comment:
```suggestion
          {isLoggedIn && isLoginModalActive ? (
```
라고 써놓고 보니 다시 isLoginActive의 역할을 잘 모르겠어요...

Reply:
한 state으로 활용하다보니 꼬여있어서 수정하고 올리겠습니다~^^